spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -1148,6 +1148,91 @@
             error_type: Faraday::NilStatusError,
           )
         end
+
+        context 'when a user was flagged with fraud_pending_reason' do
+          before(:each) do
+            pending_enrollment.profile.update!(fraud_pending_reason: 'threatmetrix_review')
+            stub_request_passed_proofing_results
+          end
+
+          it 'updates the enrollment' do
```

Comment:
This one example happens whether the flag is true or false, because it happens before we look at fraud statuses. It feels a little lonely out here though.